client/components/OrderHistory.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {getOrders} from '../store/product'
+import {Link} from 'react-router-dom'
+
+class OrderHistory extends Component {
+  constructor() {
+    super()
+    this.state = {}
+  }
+
+  componentDidMount = async () => {
+    await this.props.getUserOrders(this.props.match.params.userId)
+  }
+
+  render() {
+    const orders = this.props.items
+
+    const orderHistory = orders.length ? (
+      orders.map(order => (
+        <tbody key={order.id}>
+          <tr>
+            <h5>Order #: {order.id}</h5>
+          </tr>
+          <tr>Order Date: {order.createdAt.slice(0, 10)}</tr>
+          <tr>Order Total: ${(order.total / 100).toFixed(2)}</tr>
+          {order.products.map(item => (
+            <tr className="cart-item" key={item.name}>
+              <td>
+                <Link to={`/products/${item.order_items.productId}`}>
+                  {item.order_items.quantity} {item.name}
+                </Link>
+              </td>
+              <td>${(item.price / 100).toFixed(2)}/each</td>
+            </tr>
```

Comment:
We have a shipping and tax price on the order though we don't save them anywhere on the database so I am not sure how to reflect this on the order history - we could hard code it but we should retrieve from the database